Recipes/CGC/apps_copyFromMyProject.ipynb

```diff
@@ -90,7 +90,7 @@
     "# [USER INPUT] Set project name; project (p_) and file (f_) indices here:\n",
     "source_project_name = 'Keep on Smiling'\n",
     "my_project_name = 'Life is Beautiful'\n",
-    "app_id = 'jack_digi/keep-on-smiling/TorrentMappingAlignmentProgramMap4/0'\n",
```

Comment:
So the bindings have fixed the issue of not defaulting to "latest" if not revision is specified?